airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -220,10 +255,11 @@ def consume_logs(*, since_time: Optional[DateTime] = None, follow: bool = True)
                     ),
                     follow=follow,
                 )
-                for raw_line in logs:
-                    line = raw_line.decode('utf-8', errors="backslashreplace")
-                    timestamp, message = self.parse_log_line(line)
-                    self.log.info(message)
```

Comment:
Why do you remove it?
This was just added in https://github.com/apache/airflow/pull/23301/files

Reply:
I didnt. It is happening in `log_iterable`. I just moved the new kwarg (`, errors="backslashreplace"`) which was added by the pr you linked